docs/tutorials/access_data/framedata.md

```diff
@@ -37,8 +38,13 @@ config = runtime.set_configuration(config)
 ```
 ## Working with `AvailableData` objects
 
-During Acquisition, the `AvailableData` object is the streaming interface, and this class has a `frames` method which iterates over the `VideoFrame` objects in `AvailableData`. Once we start acquisition, we'll utilize this iterator method to list the frames. To increase the likelihood of `AvailableData` containing data, we'll utilize the time python package to introduce a delay before we create our `AvailableData` object
+During Acquisition, the `AvailableData` object is the streaming interface. By calling `get_available_data`, you create an `AvailableDataContext` object which manages memory by deleting the `AvailableData` object when it is no longer in use to free up memory. We can create an `AvailableData` object by using a `with` statement, and work with the `AvailableData` object while it exists inside of the `with` loop. 
```

Comment:
I wouldn't worry about naming the `AvailableDataContext` object. I would just tell users how to get the data and that the data is invalidated after exiting the `with` block, so they should make a copy if they want to do something nontrivial with it.